src/pal/pal_open_enclave.h

```diff
@@ -39,7 +39,8 @@ namespace snmalloc
      */
     static constexpr uint64_t pal_features = 0;
 
-    static constexpr size_t page_size = 0x1000;
+    static constexpr size_t page_size =
+      Aal::aal_name == Sparc ? Aal::smallest_page_size : 0x1000;

```

Comment:
I don't believe OpenEnclave supports SPARC, so this change looks a bit odd.

Reply:
Right I was testing under qemu and led to this code path

Reply:
```suggestion
    static constexpr size_t page_size = Aal::smallest_page_size;
```
Does this work here?